src/qt/sendcoinsdialog.cpp

```diff
@@ -502,19 +502,32 @@ void SendCoinsDialog::sendButtonClicked([[maybe_unused]] bool checked)
     }
 
     bool send_failure = false;
-    if (retval == QMessageBox::Save) {
+    if (retval == QMessageBox::Save) { // Create Unsigned clicked
         CMutableTransaction mtx = CMutableTransaction{*(m_current_transaction->getWtx())};
         PartiallySignedTransaction psbtx(mtx);
+        // Copy PSBT to clipboard and offer to save
+        presentPSBT(psbtx);
+    } else { // Send clicked
+        assert(!model->wallet().privateKeysDisabled() || model->wallet().hasExternalSigner());
         bool complete = false;
```

Comment:
a49bb0c8328d1e0ecf0ee80916594ff260f79613

I think it makes more sense to refactor this to:
```cpp
bool broadcast = true;
if (model->wallet().hasExternalSigner()) {
    // ...
    bool complete;
    send_failure = !signWithExternalSigner(psbtx, mtx, complete);
    if (complete) {
        // ...
    } else {
        // ...
        broadcast = false
    }
}
if (broadcast) {
    // ...
}